patches/patches.md

```diff
@@ -62,20 +62,6 @@ Created on **13/12/2019**
 #### Reason:
 - Change type export.
 
-
-### native-base+2.13.13
-Created on **30/07/2020**
-
-#### Reason:
```

Comment:
We also have to remove the new `native-base+2.15.2` section in this file. Am I right?

More generally I see all this  `native-base` word references:

```
./patches/patches.md:9:### native-base+2.15.2
./patches/patches.md:14:Remove this patch after removing `native-base`.
./README.md:560:We have been gradually rolling out a new library of custom components since Q1 2023. This library uses the latest React Native APIs and replaces the legacy `native-base` library, which was stuck at `v2.x`.
./ts/theme/variables.ts:2: * This file imports all the variables defined inside the native-base material
./ts/theme/variables.ts:48:  // Inputs (after removing `native-base` dependency)
```

What do we have to remove and what do we have to leave or update? What do you think?

Reply:
@shadowsheep1 Removed the reference from `patches.md` in this commit → https://github.com/pagopa/io-app/pull/5558/commits/d98f515f2d0898226ea5e7472a61e6937914942f. Other mentions are there for future reference. In particular, the `variables.ts` file is the theme file that is still partially used in the codebase and will be removed once the migration to the new DS is complete.